### ISSUE TITLE
Text-only signal status

### ISSUE THREAD

**Issue by ryanfeeley** [mozilla/MozStumbler#1060]
Until we get a beautiful set of icons, text-only (Droid Sans I assume) is proposed for now.

![text-only](https://cloud.githubusercontent.com/assets/68213/4723862/7a5f5f6e-594b-11e4-81c2-4913aec7f350.png)

When signal is not available, colour can be grey; when down, red, etc.

**Comment by garvankeeley** [comment]
Agreed, this looks really clean, the matching blue is slick. We avoided text on the toolbar because of translations. This complicates the layout quite a bit. (I am guessing only REPORTS needs translation?)

Implementation notes:
- For reports, we use 5 digits of spacing to hold a large range of values.
- We have a dash in WI-FI throughout our use in the product

**Comment by Djfe** [comment]
CELL needs to be translated, too.

EDIT: If we wait for 1.1 before changing this, then we could wait for new icons, too.

**Comment by ryanfeeley** [comment]
If localization is not surprisingly bad, and you can manage the layouts, I actually think that text performs better in this context. Fixes and new state:
![text-only-off](https://cloud.githubusercontent.com/assets/68213/4724875/9d14d206-5954-11e4-98b7-f2028bb375e7.png)
![text-only-on](https://cloud.githubusercontent.com/assets/68213/4724876/9d1ec036-5954-11e4-8249-7d8b37013520.png)

**Comment by rodmoreno** [comment]
I like so much, no icons, the app looks clean and in line with current design trends, aka, flat

**Comment by crankycoder** [comment]
I'm indifferent to icons vs text.  

In either case, the meaning of what the text or the icons mean is not clear to the user without some documentation.   So - let's get some actual formal documentation written down for users so that they understand what the number for each of the 4 things means.

**Comment by garvankeeley** [comment]
I prefer door hangar app intros to formal docs. So that when I first use an app, views hang off the UI elements explaining their purpose. In settings we would have 'Show first run screens again' or something of that nature.

**Comment by Djfe** [comment]
btw. we could move the map legend somewhere else then or/and add an explanation for the green icon, too or else people (new users) might get confused

I think the clean look is nice but icons work as well and might be even better (no translation needed)

we could maybe even offer a real tour through the app in which you are guided how to make geofenced areas and what they are (this is for later on, if we have geofences again)
